internal/controller/humiocluster_controller.go

```diff
@@ -333,6 +339,47 @@ func (r *HumioClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	return r.updateStatus(ctx, r.Client.Status(), hc, statusOptions().withState(hc.Status.State).withMessage(""))
 }
 
+// ensurePdfRenderServiceTLS ensures the pdf-render-service resource exists and has TLS config matching the HumioCluster
+func (r *HumioClusterReconciler) ensurePdfRenderServiceTLS(ctx context.Context, hc *humiov1alpha1.HumioCluster) error {
+	pdfService := &humiov1alpha1.HumioPdfRenderService{}
+	err := r.Get(ctx, types.NamespacedName{
+		Namespace: hc.Namespace,
+		Name:      hc.Name + "-pdf-render-service",
+	}, pdfService)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			// Create new pdf-render-service with TLS copied from cluster
+			newPdfService := &humiov1alpha1.HumioPdfRenderService{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      hc.Name + "-pdf-render-service",
+					Namespace: hc.Namespace,
+					Labels:    map[string]string{"humio_cluster": hc.Name},
+				},
+				Spec: humiov1alpha1.HumioPdfRenderServiceSpec{
+					TLS:      hc.Spec.TLS,
+					Image:    PDFRenderServiceImage,
```

Comment:
This one seems to always set a specific value for this. How can users configure the operator to use alternative versions for the render service? We'll have to provide some mechanism so users are able to define which version they'll want to use, especially as bugs may be found we don't want to rely on having to release a new version of "humio-operator" just so that users can use a fixed version of the render service.